### PR TITLE
Removes some tests that were only incidentally working

### PR DESCRIPTION
**Pull request by langford**
It seems like there was some strangeness going on with the cond-expr tests and that cond-expr no longer works the way it once did.

Different exceptions that seem to show the format of the cond-expr is limited to search keys means the tests themselves were asking for impossible item-updates.

Additionally, this sets up and tears down each test individually instead of working on data from the prior test. This makes adding additional tests (as I did to test variable case) more possible.

### CODE REVIEW

File: test/taoensso/faraday/tests/main.clj [hunk 1]
```diff
@@ -132,75 +132,81 @@
...
+     (far/delete-item *client-opts* ttable {:id 10})
+     (far/put-item *client-opts* ttable i-wrong-name)
      (far/update-item *client-opts* ttable
        {:id 10}
        {:update-expr     "SET #name = :name"
         :expr-attr-names {"#name" "name"}
         :expr-attr-vals  {":name" "foo"}
         :return          :all-new})))
+ 
+  (expect ; Update expression support in update-item check wierd capitalization
```
Comment (ricardojmendez) [inline test/taoensso/faraday/tests/main.clj]: What do you mean by "support for weird capitalization"?

There's a minor typo, btw. :)

File: test/taoensso/faraday/tests/main.clj [hunk 2]
```diff
@@ -132,75 +132,81 @@
...
+      (far/delete-item *client-opts* ttable {:id 10})
+      (far/put-item *client-opts* ttable  abc-set)
       (far/update-item *client-opts* ttable
                        {:id 10}
                        {:update-expr     "ADD someset :toset"
                         :expr-attr-vals  {":toset" #{"d" "e"}}
                         :return          :all-new})))
 
   (expect ; Updating an existing element to a set does not duplicate it
-    {:id 10 :name "foo" :someset #{"a" "b" "c" "d" "e" "f"}}
```
Comment (ricardojmendez) [inline test/taoensso/faraday/tests/main.clj]: I see you've removed the `:name` from a  few. Can you elaborate?

File: test/taoensso/faraday/tests/main.clj (hunk 1 above)
Comment (langford) [inline test/taoensso/faraday/tests/main.clj]: biddingEnabled is not typical Clojure capitalization. The test was making sure the library was not relying on standard Clojure capitalization

File: test/taoensso/faraday/tests/main.clj (hunk 2 above)
Comment (langford) [inline test/taoensso/faraday/tests/main.clj]: It isn't an essential part of the test, so when I duplicated the test, I removed it. 

Tests are not just for testing, they are also for demonstration of an API.

If a name *is* required, that would be good to document somewhere.

**Comment by belucid** [comment]
@langford and @ricardojmendez 

I'm looking at this now. Will comment more here once I grok what this one is all about.

**Comment by kipz** [comment]
Closing because of #136 - we can always look at this again in the future.

**Pull request closed without merging**